src/client_metrics.rs

```diff
@@ -417,6 +428,7 @@ mod tests {
             "instanceId": "test-instance-id",
             "connectionId": "test-connection-id",
             "sdkVersion": "rust-1.3.0",
+            "sdkType": "backend",
```

Comment:
lowercase from serde

Reply:
So.
If I remember correctly serde's rename lowercase doesn't work on enums. I think you have to use strum
`#[strum(serialize_all = "lowercase")]`

Reply:
Ah, but I see you have tests, ignore me then.